autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -133,7 +141,7 @@ def _invoke_assistant(
             self._openai_client.beta.threads.messages.create(
                 thread_id=assistant_thread.id,
                 content=message["content"],
-                role=message["role"],
+                role="user",
```

Comment:
Thank you for the PR :) Can you please explain this change? Or better add a brief comment in the code for hardcoded parameter justifying it?

Reply:
I noticed error the assistants API only wants "user" role for messages coming to threads, did you not see this in your tests?